### ISSUE TITLE
4.8.1 missing from CocoaPods

### ISSUE THREAD

**Issue by stianjensen** [firebase/firebase-ios-sdk#726]
### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 9
  * Firebase SDK version: 4.8.1
  * Firebase Component: Core and Messaging (Auth, Core, Database, Firestore, Messaging, Storage, etc)
  * Component version: 4.0.14

#### Steps to reproduce:

What happened? How can we make the problem occur?

In order to get as close to a deterministically reproducible build as possible, we have locked our Firebase SDK dependency in our podfile to version 4.8.1 specifically. Until this weekend, this was the latest released version. Today, on Travis, builds suddenly started failing with the following error message:

```
[!] CocoaPods could not find compatible versions for pod "Firebase/Core":
  In snapshot (Podfile.lock):
    Firebase/Core (= 4.8.1)
  In Podfile:
    Firebase/Core
None of your spec sources contain a spec satisfying the dependencies: `Firebase/Core, Firebase/Core (= 4.8.1)`.
You have either:
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.
```

Note that we always run with `pod install --repo-update` so that new packages we add to our lockfile always can be installed by travis, but we lock the version numbers on them to ensure that the version we want is installed.

I took a look in the cocoapods repository today, and I see that version 4.8.1 has been deleted from there, and then 2 days later version 4.8.2 has been released. I find no mention of this in the commits or issues in this repository here, so I wonder why that happened.

Commit that deleted 4.8.1:
https://github.com/CocoaPods/Specs/commit/12c0e530ef31545e91ff2e107050e0e6d69c8ad6

**Comment by morganchen12** [comment]
Over the weekend we discovered a critical issue within the Analytics 4.0.7 SDK. We've released Firebase v4.8.2, which contains a fix for the analytics issue and no other changes.

Deleting a pod is something that for the most part is highly frowned upon by CocoaPods, but due to the persistent nature of Analytics (bugs in data reporting have lasting effects across versions), we decided it was worth it to not allow the bugged version of Analytics to be installed by people who hadn't updated their CocoaPods specs at the cost of some inconvenience to our users.

We apologize for any difficulties this may have caused. If you have trouble with upgrading from 4.8.1 to 4.8.2, comment here and we'll help you out.
